modules/index-service/src/main/java/org/opencastproject/index/service/impl/IndexServiceImpl.java

```diff
@@ -2143,4 +2166,40 @@ public boolean isWorkflowActive(String workflowState) {
             || WorkflowState.PAUSED.toString().equals(workflowState);
   }
 
+  private void checkAcceptedTypes(String assetUploadId, MediaType mediaType) {
```

Comment:
I feel kinda bad about the IndexService becoming even longer than it is now... what would you think about moving this into its own class? Too weird?

Reply:
I just took a look and there already is `RequestUtils` and I think this method fits in there quite well. So I moved it there.